### ISSUE TITLE
Easier theme changing / Font size / color 

### ISSUE THREAD

**Issue by mstdn** [tootsuite/mastodon#14210]
Meow,

On Mastodon I switch many times between themes on a day. It depends what's best for my eyes at a certain time but to be honest it's a pain on Mastodon. I would like to purpose a comparison with how the birdsite does this without leaving the page.

### Pitch

Can't we also have such a option without leaving the web UI? See Twitter screenshots.

<img width="289" alt="Schermafbeelding 2020-07-04 om 19 18 48" src="https://user-images.githubusercontent.com/57602062/86517752-aa677700-be2b-11ea-869d-b894db7a484a.png">
<img width="604" alt="Schermafbeelding 2020-07-04 om 19 19 01" src="https://user-images.githubusercontent.com/57602062/86517754-ab000d80-be2b-11ea-83ba-2d1c5d44a933.png">

### Motivation

People with reading problems and users who change themes a lot I guess.

**Comment by hirusi** [comment]
Also relevant:https://github.com/tootsuite/mastodon/issues/10976

**Comment by mstdn** [comment]
> Also relevant:#10976

Yes, but I want to even a step further with editable font size and perhaps some extra colour variants
